scripts/performance/configs/qwen_vl/qwen3_vl_llm_pretrain.py

```diff
@@ -0,0 +1,259 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
I am not sure we want to add one entrypoint script for each model. can we generalize to vlm not limited to qwen3 as we do in https://github.com/NVIDIA-NeMo/Megatron-Bridge/blob/main/scripts/performance/run_script.py .